cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,52 @@
-describe('', () => {
-  before(() => {
+describe('Article flow', () => {
+  let article;
 
+  beforeEach(() => {
+    cy.visit('/');
+    cy.login('gitepi3626@vaband.com', 'pXTDbz8429');
```

Comment:
You have set default values of email and password, you don't have to write them here again.